patches/disable_cqe_compression.patch

```diff
@@ -0,0 +1,60 @@
+Disable cqe_compression at the driver code level
+
+From: dennisafa <dennis.afa@gmail.com>
+
+
+---
+ drivers/net/mlx5/linux/mlx5_os.c |   12 +-----------
+ drivers/net/mlx5/mlx5.c          |    2 +-
+ 2 files changed, 2 insertions(+), 12 deletions(-)
+
+diff --git a/drivers/net/mlx5/linux/mlx5_os.c b/drivers/net/mlx5/linux/mlx5_os.c
+index 69123e12c..3ccd9016d 100644
+--- a/drivers/net/mlx5/linux/mlx5_os.c
++++ b/drivers/net/mlx5/linux/mlx5_os.c
+@@ -509,7 +509,6 @@ mlx5_dev_spawn(struct rte_device *dpdk_dev,
+ 	int err = 0;
+ 	unsigned int hw_padding = 0;
+ 	unsigned int mps;
+-	unsigned int cqe_comp;
+ 	unsigned int cqe_pad = 0;
+ 	unsigned int tunnel_en = 0;
+ 	unsigned int mpls_en = 0;
+@@ -690,12 +689,7 @@ mlx5_dev_spawn(struct rte_device *dpdk_dev,
+ 			mprq_caps.max_single_wqe_log_num_of_strides;
+ 	}
+ #endif
+-	if (RTE_CACHE_LINE_SIZE == 128 &&
+-	    !(dv_attr.flags & MLX5DV_CONTEXT_FLAGS_CQE_128B_COMP))
```

Comment:
Totally taking your word for the meaning and applicability of these other attributes.